### ISSUE TITLE
Array to string conversion exception with db:info command

### ISSUE THREAD

**Issue by guillaume-netapsys** [netz98/n98-magerun2#268]
Hello,

I'm currently trying to execute this command with my project : 

`n98-magerun2 --skip-root-check --ansi db:info --format json`

But it returns the following error : 

`[Exception] 12:15:09   Notice: Array to string conversion in xxx/n98-magerun2/src/N98/Magento/Command/Database/InfoCommand.php on line 48`

It appears to be a problem with Magento2 `app/etc/env.php` file.

I need to use the following configuration for one of my module : 

```
'db' =>
  array (
    'table_prefix' => '',
    'connection' =>
    array (
      'default' =>
      array (
        'host' => 'xxx',
        'dbname' => 'xxx',
        'username' => 'xxx',
        'password' => 'xxx',
        'model' => 'mysql4',
        'engine' => 'innodb',
        'initStatements' => 'SET NAMES utf8;',
        'active' => '1',
        'driver_options' => array(PDO::MYSQL_ATTR_LOCAL_INFILE => true),
      ),
    ),
  )
```

It appears that the command is not handling an array as a value of a key in the "default" array.

Thank you.

**Comment by tkn98** [comment]
Can you also please try with the development (unstable) version? This report rings a bell and it might be this is fixed already.

**Comment by guillaume-netapsys** [comment]
Hello,

I've tried to switch to development version and it's working in those conditions.
Can you tell me when this feature/fix will be included in the stable version ?

Thank you.

**Comment by ktomk** [comment]
@guillaume-netapsys: I plan to make a release today.

**Comment by ktomk** [comment]
Fix for issue is in ea2ce39b02adc7ed2e661e553988d0d7d667a7b9.
